google/cloud/spanner_dbapi/connection.py

```diff
@@ -41,28 +52,46 @@ def __handle_update_ddl(self, ddl_statements):
         """
         Run the list of Data Definition Language (DDL) statements on the underlying
         database. Each DDL statement MUST NOT contain a semicolon.
-        Args:
-            ddl_statements: a list of DDL statements, each without a semicolon.
-        Returns:
-            google.api_core.operation.Operation.result()
+
+        :type ddl_statements: list
+        :param ddl_statements: a list of DDL statements, each without a semicolon.
+
+        :rtype: :func:`google.api_core.operation.Operation.result()`
+        :returns: result of the operation
         """
         self._raise_if_closed()
         # Synchronously wait on the operation's completion.
         return self._dbhandle.update_ddl(ddl_statements).result()
 
     def read_snapshot(self):
+        """Returns Data Base snapshot if connection is open.
+
+        :rtype: :class:`Snapshot`
+        :returns: database snapshot.
+        """
         self._raise_if_closed()
         return self._dbhandle.snapshot()
 
     def in_transaction(self, fn, *args, **kwargs):
+        """Runs :func:`spanner_v1.Database.run_in_transaction` inside open connection.
+
+        :rtype: Any
+        :returns: result of transaction running.
+        """
         self._raise_if_closed()
         return self._dbhandle.run_in_transaction(fn, *args, **kwargs)
 
     def append_ddl_statement(self, ddl_statement):
+        """Appends DDL statement."""
         self._raise_if_closed()
         self._ddl_statements.append(ddl_statement)
 
     def run_prior_DDL_statements(self):
+        """Runs prior DDL statements.
+
+        :rtype: :func:`__handle_update_ddl()`
```

Comment:
As before, the return is a `google.protobuf.Message` which is of a value, not reference type. Check out the `Operation.result()` link given above.